src/services/pseudonym/hooks/index.js

```diff
@@ -1,5 +1,6 @@
 const { authenticate } = require('@feathersjs/authentication');
 const errors = require('@feathersjs/errors');
+const { disallow } = require('feathers-hooks-common');
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Unable to resolve path to module 'feathers-hooks-common'.](https://app.codacy.com/manual/schul-cloud/schulcloud-server/pullRequest?prid=4681704)